### ISSUE TITLE
[BUG] PolyStrips: Unhandled Exception Caught: Exception caught

### ISSUE THREAD

**Issue by LLKoder** [CGCookie/retopoflow#979]
Issue Template
-----------------------------------

**Issue:**

A crash occurs when I'm changing the "Cut Count".

![RetopoFlow_screenshot](https://user-images.githubusercontent.com/50442676/112081500-7c031500-8bcf-11eb-96d8-49956d4a86b3.png)

**How to Reproduce:**

1, Open Blender 2.92 (portable if it is possible).
2. The Cube is created as default.
3. Run RF.
4. Create a line of PolyStrips.
5. Change "Cut Count".
6. Get the crash.

Addition:
7. "Scale Falloff" is working but it doesn't apply immediately. I have to redraw the line.

Automatically Generated Info
-----------------------------------

```
Environment:

- RetopoFlow: 3.1.0
- CG Cookie built
- Blender: 2.92.0 master 2021-02-24
- Platform: Windows, 10, 10.0.19041, AMD64, Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
- GPU: NVIDIA Corporation, GeForce GTX 1050/PCIe/SSE2, 4.5.0 NVIDIA 461.72, 4.50 NVIDIA
- Timestamp: 2021-03-23 11:59:09.734014
- Undo: change segment count

Error Hash: 13ade57af8d1e21936464ad89eff5f9a

Trace:

- EXCEPTION (<class 'AttributeError'>): 'PolyStrips' object has no attribute 'count_data'
-     C:\Users\***\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\RetopoFlow\retopoflow\rftool_polystrips\polystrips_ops.py
- 000 0404:change_count() self.count_data['nfaces'].clear()
-     C:\Users\***\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\RetopoFlow\retopoflow\rftool.py
- 001 0158:wrapper() ret = fn(*args, **kwargs)
-     C:\Users\***\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\RetopoFlow\retopoflow\rftool_polystrips\polystrips_props.py
- 002 0055:var_cut_count() self.change_count(count=v)
-     <string>
- 003 0001:<module>() 
-     .../common\boundvar.py
- 004 0095:set_value() exec(f'{self._value_str} = {self._pre_wrap}{value}{self._post_wrap}', self._f_globals, self._f_locals)
- 005 0101:value() def value(self, value): self.set_value(value)
-     .../common\ui_properties.py
- 006 0955:value() if self._value_bound:   self._value.value = v
-     .../common\ui_elements.py
- 007 0490:blur() self.value = data['text']
-     .../common\ui_core.py
- 008 2033:_fire_event() cb(details)
- 009 2039:_fire_event() raise e
- 010 2070:dispatch_event() self._fire_event(event, ui_event)
-     .../common\ui_document.py
- 011 0595:blur() self._focus.dispatch_event('on_blur')
-     .../common\ui_properties.py
- 012 0493:blur() self._document.blur()
-     .../common\ui_elements.py
- 013 0516:keypress() self.blur()
-     .../common\ui_core.py
- 014 2033:_fire_event() cb(details)
- 015 2039:_fire_event() raise e
- 016 2070:dispatch_event() self._fire_event(event, ui_event)
-     .../common\ui_document.py
- 017 0366:handle_keypress() ui_element.dispatch_event('on_keypress', key=pressed)
- 018 0642:focus_main() self.handle_keypress()
-     .../common\fsm.py
- 019 0058:run() return fn(*args, **kwargs)
```

**Comment by vxlcoder** [comment]
thanks for the report, @LLKoder.

this should now be fixed in commit 8bf0426, and the fix will be in the next release.

**Comment by jlampel** [comment]
Confirmed fixed!
